libbeat/idxmgmt/std.go

```diff
@@ -213,19 +204,18 @@ func (m *indexManager) VerifySetup(loadTemplate, loadILM LoadMode) (bool, string
 		m.support.templateCfg.Overwrite, loadTemplate)
 
 	if ilmComponent.load && !templateComponent.load {
-		return false, "Loading ILM policy and write alias without loading template " +
-			"is not recommended. Check your configuration."
+		return false, "Loading ILM policy without loading template is not recommended. Check your configuration."
 	}
 
 	if templateComponent.load && !ilmComponent.load && ilmComponent.enabled {
 		return false, "Loading template with ILM settings whithout loading ILM " +
-			"policy and alias can lead to issues and is not recommended. " +
+			"policy can lead to issues and is not recommended. " +
```

Comment:
As a follow up: This log message and the above, we likely could be more specific on which config should be checked exactly to make it easier for users to find it.